recipes/cram/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cram" %}
+{% set version = "0.7" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "md5" %}
```

Comment:
sha256 is preferred. https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L7